src/plugins/media/nodeviews/mediaNodeView/media.tsx

```diff
@@ -84,11 +102,20 @@ export class MediaNode extends Component<MediaNodeProps, MediaNodeState> {
   async componentDidMount() {
     this.handleNewNode(this.props)
 
-    const { contextIdentifierProvider } = this.props
+    const { contextIdentifierProvider, collabEditProvider } = this.props
+
     this.setState({
-      contextIdentifierProvider: await contextIdentifierProvider
+      contextIdentifierProvider: await contextIdentifierProvider,
+      collabEditProvider: await collabEditProvider
     })
 
+    this.state.collabEditProvider?.serviceClient.realtime.subscribe(
+      RealTimeEvent.Updated,
+      CozyDoctypes.Files,
+      () => this.setState({ isLoading: false })
+    )
```

Comment:
Here you subscribe to all the updated events for `io.cozy.files` and the callback is just setting the isLoading. It's weird. 

Shouldn't we subscribe to the needed id and update the node view with the new value?

Reply:
I think it was missing an info from the back-end at this point. It is now updated